### PR TITLE
Template

### PR DESCRIPTION
**Pull request by C0deJack**
First pull request for the main template

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    <link rel="stylesheet" href="../normalize.css" type="text/css" media="screen">
+    <link rel="stylesheet" type="text/css" href="styles.css">
+    <title>Jack Wood-Pearce</title>
+</head>
+<body>
+<header>
+    <h1>Jack Wood-Pearce</h1>
+</header>
+<section>
+
```
Comment (obwansan) [inline about.html:13]: empty section?

File: index.html [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+<head>
+    <link rel="stylesheet" href="normalize.css" type="text/css" media="screen">
+    <link rel="stylesheet" type="text/css" href="styles.css">
+    <title>Jack Wood-Pearce</title>
+</head>
+<body>
+<header id="home">
+    <a href="#home"><h1>Jack Wood-Pearce</h1></a>
+    <nav>
+        <a href="#portfolio"><div class="nav-1">PORTFOLIO</div></a>
```
Comment (obwansan) [inline index.html]: <div> (block-element) in <a> (inline-element) is invalid CSS

File: index.html [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+        <h3>07777777777</h3>
+        <h3>jackwoodpearce@gmail.com</h3>
+    </div>
+    <a href="#home" class="side-tab"><img src="assets/uparrow.png"></a>
+</header>
+<section>
+    <video autoplay muted loop class="cloud-video">
+        <source src="assets/cloud.mp4" type="video/mp4">
+    </video>
+    <div id="about" class="about">
```
Comment (obwansan) [inline index.html]: Don't know if there's a best practice thing about IDs and classes having the same name?

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+    transition-timing-function: ease-in-out;
+    background-color: #ABB2B9;
+}
+
+[class|=nav] a {
+    text-decoration: none;
+    color: #FFFFFF;
+}
+
+.nav-1 {
```
Comment (obwansan) [inline styles.css]: Maybe tag names could be more descriptive? Are there 3 navbars or 3 sections within 1 navbar etc?

File: about.html (hunk 1 above)
Comment (C0deJack) [reply about.html:13]: place holder that's now being used

File: index.html (hunk 2 above)
Comment (C0deJack) [reply index.html]: its a work around to make the whole of the navigation buttons clickable

File: index.html (hunk 3 above)
Comment (C0deJack) [reply index.html]: cool changing this

File: styles.css (hunk 4 above)
Comment (C0deJack) [reply styles.css]: sorted

**Comment by obwansan** [comment]
All good - passed code review

**Pull request merged**